model/lifecycle.go

```diff
@@ -845,9 +851,26 @@ func TryMarkPatchBuildFinished(b *build.Build, finishTime time.Time, updates *St
 	return nil
 }
 
+func getTaskCreateTime(project *Project, v *version.Version) (*time.Time, error) {
```

Comment:
why return a pointer to time.Time? It seems more complicated than just return the concrete value.

Reply:
Right, I used a pointer to return nil, I have to get used to returning empty structs. Done.